src/libs/Performance.tsx

```diff
@@ -197,14 +235,12 @@ if (Metrics.canCapturePerformanceMetrics()) {
 
     /**
      * A HOC that captures render timings of the Wrapped component
-     * @param {object} config
-     * @param {string} config.id
-     * @returns {function(React.Component): React.FunctionComponent}
      */
     Performance.withRenderTrace =
-        ({id}) =>
-        (WrappedComponent) => {
-            const WithRenderTrace = forwardRef((props, ref) => (
+        ({id}: WrappedComponentConfig) =>
+        // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
@VickyStash why this is added?

Reply:
@kubabutkiewicz Because WrappedComponent is not in camel case
![image](https://github.com/Expensify/App/assets/23176449/5b2a18a0-5639-4f45-843a-1ba64f56ac0e)